frontend/src/shared/helpers.tsx

```diff
@@ -517,7 +517,13 @@ export const CohortsListColumns: ColDef[] = [
     sortable: false,
   },
   {
-    headerName: "Cohort Complete Date",
+    headerName: "Delivery Date",
+    valueGetter: ({ data }) =>
+      data["hasCohortCompleteCohortCompletes"]?.slice(-1)[0]?.date,
+    sortable: false,
+  },
+  {
+    headerName: "Complete Date",
```

Comment:
Renamed this from `Cohort Complete Date` to `Complete Date`. Followed this same logic with `Delivery Date` above. I think the `Cohort` part is already implied given each record/row is a cohort.